src/java/fr/paris/lutece/plugins/genericattributes/service/entrytype/AbstractEntryTypeDate.java

```diff
@@ -208,7 +204,7 @@ public GenericAttributeError getResponseData( Entry entry, HttpServletRequest re
     @Override
     public String getResponseValueForExport( Entry entry, HttpServletRequest request, Response response, Locale locale )
     {
-        return response.getResponseValue( );
+       return getResponseValueForRecap( entry, request, response, locale );
```

Comment:
This value is to be used by a machine, in contrast to the recap value which is meant to be presented to the user.
Maybe the format should be separate, and configurable in forms.properties (see `forms.export.formResponse.form.date.creation.format`property